app/models/auctify/sale/auction.rb

```diff
@@ -249,6 +263,23 @@ def auction_prolonging_limit_in_seconds
         pack&.auction_prolonging_limit_in_seconds || Auctify.configuration.auction_prolonging_limit_in_seconds
       end
 
+      def close_manually(by:, price_check:)
+        if manually_closed_at.nil? && update(manually_closed_at: Time.current, manually_closed_by: by, manually_closed_price_check: price_check)
+          Auctify::BiddingCloserJob.perform_later(auction_id: id)
```

Comment:
Tady mi přijde zbytečné to honit přes Job. Rovnou bych dal `auction.close_bidding!`